src/components/chrome/index.tsx

```diff
@@ -60,6 +64,15 @@ const AppLogo: FunctionComponent<{ onClick: () => void }> = ({ onClick }) => (
  */
 const Chrome: FunctionComponent = ({ children }) => {
   const router = useRouter();
+  const { colorMode } = useProvider();
+
+  const emotionCache = createCache({
```

Comment:
I tried to heavily annotate the source of this project, so that developers could understand what the different bits and pieces were for. Can we add a comment explaining what this cache is all about? I'd find it helpful 😅